### DISCUSSION TITLE
How do I disable the first boot startup scripts?

### DISCUSSION THREAD

**Discussion by lunasophia** [Plebian-Linux/plebian-website#7] [Q&A] [answered]
I have booted the board ~~many, many times while testing things~~ and would like to disable the first boot startup scripts. It seems they're part of the initramfs machinery, but I'd like to be sure before I start making changes so I don't break anything. Some guidance would be appreciated! It'd be great to have this in the official documentation.

**Comment by CounterPillow** [comment, marked as answer]
Disabling the resize root hook should be as easy as `sudo chmod -x /etc/initramfs-tools/hooks/resizerootfs && sudo update-initramfs -u` but I'll have to double check that before I put it in the documentation.

The openssh key generation is a systemd unit that only runs if the host keys are missing, so that doesn't need to be disabled since it won't run.
